import-export-cli/integration/testutils/operationPolicy_testUtils.go

```diff
@@ -0,0 +1,552 @@
+	var output string
+	var err error
+	limit := TestAPIPolicyLimit
+
+	output, err = base.Execute(t, "get", "policies", "api", "-e", args.SrcAPIM.EnvName, "--all", "--limit", limit, "--format", "jsonArray", "-k", "--verbose")
+
+	return output, err
+}
+
+// get the API policy list apictl output
+func listAPIPoliciesWithAllFlag(t *testing.T, args *PolicyImportExportTestArgs) (string, error) {
+	var output string
+	var err error
+
+	output, err = base.Execute(t, "get", "policies", "api", "-e", args.SrcAPIM.EnvName, "--all", "--format", "jsonArray", "-k", "--verbose")
+
+	return output, err
+}
+
+// Checks whether apictl output contains all the available api policy UUIDs
+func validateListAPIPoliciesEqual(t *testing.T, apiPoliciesListOutput string, apiPoliciesList *apim.APIPoliciesList) {
+	unmatchedCount := apiPoliciesList.Count
+
+	for _, policy := range apiPoliciesList.List {
+		// If the output string contains the same Policy ID, then decrement the count
+		assert.Truef(t, strings.Contains(apiPoliciesListOutput, policy.Id), "APIPoliciesListFromCtl: "+apiPoliciesListOutput+
+			" , does not contain policy.id: "+policy.Id)
+		unmatchedCount--
+	}
+	// Count == 0 means that all the policies from apiPoliciesList were in apiPoliciesListOutput
+	assert.Equal(t, 0, unmatchedCount, "API policies lists are not equal")
+}
+
+// Checks whether apictl output contains all the available api policy UUIDs
+func validateListAPIPoliciesEqualWithLimit(t *testing.T, apiPoliciesListOutput []apim.PolicySpecData, apiPoliciesList *apim.APIPoliciesList, limit int) {
+	assert.LessOrEqual(t, len(apiPoliciesListOutput), limit, "API Policy list output size is not less than or equivalent with Limit")
+	assert.Equal(t, apiPoliciesList.Count, len(apiPoliciesListOutput), "API policies list sizes are not equal")
+
+	for i, policy := range apiPoliciesList.List {
+		// fmt.Println("Policy ID It: ", apiPoliciesListOutput[i].Id)
+		assert.Equal(t, apiPoliciesListOutput[i].Id, policy.Id, "API Policies are not equal")
+	}
+}
+
+// Checks whether apictl output has all policies
+func validateListAPIPoliciesEqualWithAllFlag(t *testing.T, apiPoliciesListOutput []apim.PolicySpecData, apiPoliciesList *apim.APIPoliciesList) {
+	assert.Equal(t, apiPoliciesList.Count, len(apiPoliciesListOutput), "API policies list sizes are not equal")
+
+	for i, policy := range apiPoliciesList.List {
+		// fmt.Println("Policy ID It: ", apiPoliciesListOutput[i].Id)
+		assert.Equal(t, apiPoliciesListOutput[i].Id, policy.Id, "API Policies are not equal")
+	}
+}
+
+func readAPIPolicyDefinition(t *testing.T, path string) apim.PolicySpecData {
+
+	// Read the file in the path
+	sampleData, err := ioutil.ReadFile(path)
+	if err != nil {
+		t.Error(err)
+	}
+
+	// Extract the content to a structure
+	sampleContent := apim.APIPolicyFile{}
+	err = yaml.Unmarshal(sampleData, &sampleContent)
+	if err != nil {
+		t.Error(err)
```

Comment:
I see that in some places we have given a `t.Error` for issues with unmarshalling and in other places we've used `t.Fatal`. Fatal will be followed by an `os.Exit(1)` command. Can we double check on this and make it behave in a similar manner?

Reply:
Yeah, we have to fail the test from the error point. No need of proceeding ahead and checking other errors if exist inside the function right? @wasuradananjith If so, `t.Fatal` is the ideal one as @ashera96 has mentioned

Reply:
+1 to use t.Fatal

Reply:
Fixed by [6fa5d86](https://github.com/wso2/product-apim-tooling/pull/925/commits/6fa5d860248f4c67f4525b5f3a25a7e212feb57d)